setup.py

```diff
@@ -30,6 +30,8 @@
             'pytest',
             'pytest-cov',
             'webtest',
+            'tox',
+            'flake8'
         ],
```

Comment:
adding `flake8` here does not seem to be necessary, as it gets installed by tox, see this configuration:
```
[testenv:pep8]
basepython = python3
deps = flake8
commands = flake8 morepath_wiki setup.py
```

`deps` means install... a package

Adding `tox` here is probably ok. I have installed it globally, as every project I maintain uses tox, but if that helps you - go for it.

Reply:
Done